data/processors/export.py

```diff
@@ -41,6 +41,19 @@ def unlocalise(value: str | list[Any] | dict[str, Any]) -> Any:
     raise ValueError(f"Unhandled type {type(value)}")
 
 
+def normalise_id(_id: str) -> str:
+    """Remove leading zeros from all point-separated parts of input string"""
+    parts = _id.split(".")
+    for i in range(0, len(parts)):
+        parts[i] = parts[i].lstrip("0")
+        if not parts[i]:
+            parts[i] = "0"
+    result: str = ""
+    for part in parts:
+        result += part
+    return ".".join(parts)
```

Comment:
This might be nice to know:
Iterating with indexes in python is really bad performance wise.
For cases like this, list comprehensions are a bit more readable and a lot more performant.
(not like performance matters here, but )

```suggestion
    parts = [part.lstrip("0") or "0" for part in _id.split(".")]
    return ".".join(parts)
```